src/main/navigation/navigation.c

```diff
@@ -2775,7 +2775,7 @@ bool navigationRequiresTurnAssistance(void)
     const navigationFSMStateFlags_t currentState = navGetStateFlags(posControl.navState);
     if (STATE(FIXED_WING)) {
         // For airplanes turn assistant is always required when controlling position
-        return (currentState & NAV_CTL_POS);
+        return (currentState & (NAV_CTL_POS | NAV_CTL_ALT));
```

Comment:
I think it would be better for planes/wings. When trying to maintain altitude it is required to coordinate roll/pitch/yaw. It is not strictly necessary but it would probably give better experience "out of the box" meaning without the user enabling it manually.